DICOM/src/Microsoft.Health.DeID.SharedLib/Models/CryptoHashType.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Health.DeID.SharedLib.Models
+{
+    public enum CryptoHashType
```

Comment:
🟢 RECOMMENDATION

`CryptoHashAlgorithm` or simpler: `HashAlgorithm`
Actually their implementation is in System.Security.Cryptography.**Algorithms**.dll

Reply:
+1 [Reference](https://docs.microsoft.com/en-us/dotnet/api/system.security.authentication.hashalgorithmtype?view=net-5.0)

Reply:
Fixed it.